docs/platform/reference/service-ip-address.rst

```diff
@@ -19,14 +19,15 @@ When a new service is being provisioned, its hostname is defined as follows:
 
 .. code::
 
-   <SERVICE_NAME>-<PROJECT_NAME>.aivencloud.com
+   <SERVICE_NAME>-<PROJECT_NAME>.*.aivencloud.com
 
 
-where:
+where
 
 * ``<SERVICE_NAME>`` is the name of the service
 * ``<PROJECT_NAME>`` is the name of the project
+* ``*`` is a variable component consisting of one or more letters of alphanumeric subdomains
```

Comment:
consisting of one or more **levels** of alphanumeric subdomains

Reply:
So it could be something.a.b.c.aivencloud.com, or it could suddenly be something.a123.afdff00.aiven-cloud.net.